internal/config/config_file.go

```diff
@@ -206,3 +215,16 @@ func findRegularFile(p string) string {
 	}
 	return ""
 }
+
+func userHomeDir() string {
+	if runtime.GOOS == "windows" {
+		home := os.Getenv("HOMEDRIVE") + os.Getenv("HOMEPATH")
```

Comment:
Did you intentionally avoid using `homedir.Dir()` here?

Reply:
I didn't intentional avoid using homedir.Dir. I have since removed this in the latest commit